<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1399</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>512</width>
      <height>512</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>512</width>
      <height>512</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>801</width>
      <height>731</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>141</width>
       <height>401</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>双立方插值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>缩小</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>高度倍数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_small_biCubic_height"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>宽度倍数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_small_biCubic_width"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_small_biCubic">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>放大</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>高度倍数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_large_biCubic_height"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>宽度倍数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_large_biCubic_width"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_large_biCubic">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_resetAll">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>670</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>重置图像</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>141</width>
       <height>401</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>双线性插值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>缩小</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>高度倍数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_small_bilinear_height"/>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>宽度倍数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_small_bilinear_width"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_small_bilinear">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>放大</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>高度倍数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_large_bilinear_height"/>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>宽度倍数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_large_bilinear_width"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_large_bilinear">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_face">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>92</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>一键瘦脸</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_fish_eye">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>50</y>
       <width>92</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>鱼眼镜头</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>277</width>
       <height>189</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit_brightness"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_brightness">
           <property name="text">
            <string>改变亮度</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit_contrast"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_contrast">
           <property name="text">
            <string>改变对比度</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit_saturation">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_saturation">
           <property name="text">
            <string>改变饱和度</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_8" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>253</width>
       <height>134</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>高斯模糊</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_9" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>模糊程度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_gauss"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_gauss_ok">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_10" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>271</width>
       <height>217</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QWidget" name="widget_11" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="btn_gray">
           <property name="text">
            <string>转为灰度图</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_blend">
           <property name="text">
            <string>图层混合</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_colorMap">
           <property name="text">
            <string>色彩映射</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_averBlur">
           <property name="text">
            <string>均值模糊</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_eye">
           <property name="text">
            <string>眼部液化</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_12" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="btn_autoContrast">
           <property name="text">
            <string>自动调整对比度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_color_level">
           <property name="text">
            <string>色阶调整</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_reverse_color">
           <property name="text">
            <string>色彩反转</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_complementary">
           <property name="text">
            <string>补色</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_color_balance">
           <property name="text">
            <string>色彩平衡</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_13" native="true">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>420</y>
       <width>223</width>
       <height>134</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>高反差保留</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_14" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>程度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_highContrast"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_highContrast_ok">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_add" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>551</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_15" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>377</width>
       <height>180</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>图像旋转</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_16" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QWidget" name="widget_17" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>顺时针旋转</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_rotate"/>
            </item>
            <item>
             <widget class="QPushButton" name="btn_rotate_ok">
              <property name="text">
               <string>确定</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_18" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>逆时针旋转</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_rotate_r"/>
            </item>
            <item>
             <widget class="QPushButton" name="btn_rotate_r">
              <property name="text">
               <string>确定</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1399</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openImage"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openImage">
   <property name="text">
    <string>打开图像文件</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
